src/nova_renderer.cpp

```diff
@@ -792,6 +811,38 @@ namespace nova::renderer {
         }
     }
 
+    void NovaRenderer::record_rendering_static_mesh_batch(const ProceduralMeshBatch<StaticMeshRenderCommand>& batch, rhi::CommandList* cmds) {
+        const uint32_t start_index = cur_model_matrix_index;
+
+        for (const StaticMeshRenderCommand& command : batch.commands) {
+            if (command.is_visible) {
+                auto* model_matrix_buffer = builtin_buffers.at(MODEL_MATRIX_BUFFER_NAME);
+                rhi->write_data_to_buffer(&command.model_matrix,
+                    sizeof(glm::mat4),
+                    cur_model_matrix_index * sizeof(glm::mat4),
+                    model_matrix_buffer);
+                cur_model_matrix_index++;
+            }
+        }
+
+        if (start_index != cur_model_matrix_index) {
+        const auto& [vertex_buffer, index_buffer] = batch.mesh->get_buffers_for_frame(cur_frame_idx);
+            // TODO: There's probably a better way to do this
+            const std::vector<rhi::Buffer*> vertex_buffers = { vertex_buffer,
```

Comment:
`const std::vector<rhi::Buffer*> vertex_buffers(7, vertex_buffer);`

I always have to look up the decl for `std::vector::vector`...